irohad/torii/impl/command_service.cpp

```diff
@@ -19,18 +19,21 @@
 
 #include <thread>
 
-#include "backend/protobuf/transaction_responses/proto_tx_response.hpp"
-
 #include "ametsuchi/block_query.hpp"
 #include "backend/protobuf/transaction.hpp"
-#include "builders/protobuf/transaction_responses/proto_transaction_status_builder.hpp"
+#include "backend/protobuf/transaction_responses/proto_tx_response.hpp"
+// breaking sequence of this two includes will break the build
+// clang-format off
 #include "builders/protobuf/transport_builder.hpp"
+#include "builders/protobuf/transaction_sequence_builder.hpp"
```

Comment:
How about to add `#include "builders/protobuf/transport_builder.hpp"` to `transaction_sequence_builder.hpp`, so that no one ever experience that issue (I suspect developers might spend too much time to figure out why same problem arises)